ext/date/php_date.stub.php

```diff
@@ -536,76 +532,34 @@ public static function createFromInterface(DateTimeInterface $object): DateTimeI
 
 class DateTimeZone
 {
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_AFRICA
-     */
-    public const AFRICA = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_AMERICA
-     */
-    public const AMERICA = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_ANTARCTICA
-     */
-    public const ANTARCTICA = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_ARCTIC
-     */
-    public const ARCTIC = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_ASIA
-     */
-    public const ASIA = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_ATLANTIC
-     */
-    public const ATLANTIC = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_AUSTRALIA
-     */
-    public const AUSTRALIA = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_EUROPE
-     */
-    public const EUROPE = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_INDIAN
-     */
-    public const INDIAN = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_PACIFIC
-     */
-    public const PACIFIC = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_UTC
-     */
-    public const UTC = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_ALL
-     */
-    public const ALL = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_GROUP_ALL_W_BC
-     */
-    public const ALL_WITH_BC = UNKNOWN;
-    /**
-     * @var int
-     * @cvalue PHP_DATE_TIMEZONE_PER_COUNTRY
-     */
-    public const PER_COUNTRY = UNKNOWN;
+    /** @cvalue PHP_DATE_TIMEZONE_GROUP_AFRICA */
```

Comment:
Please don't make unnecessary formatting changes (just remove the `@var int` line).